packages/react/src/components/MultiSelect/MultiSelect.stories.js

```diff
@@ -65,6 +65,9 @@ export default {
     open: {
       table: { disable: true },
     },
+    readOnly: {
+      control: { type: 'boolean' },
+    },
```

Comment:
Instead of adding this here to all stories, interactive controls should only be added to the playground story args lower down in this file.